Dockerfile

```diff
@@ -3,7 +3,7 @@ WORKDIR /root/containerbuild
 
 # Only need a few of our scripts for the first few steps
 COPY ./src/cmdlib.sh ./build.sh ./deps.txt ./vmdeps.txt ./build-deps.txt /root/containerbuild/
-RUN ./build.sh configure_yum_repos
+#RUN ./build.sh configure_yum_repos
```

Comment:
Is this still necessary?

Reply:
Not sure, I'll do it again from scratch when I get the FCOS image to run in aarch64.

Reply:
Not necessary anymore. I removed it from the PR.